drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/mper.rb

```diff
@@ -0,0 +1,55 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis
+  # Utility class for MPER
+  class Mper
+    SYSTEM_ID = 'ac_hmis_mper'.freeze
+
+    # @param mper_id [String]
+    # @return [Hmis::Hud::Project, nil]
+    def find_project_by_mper(mper_id)
+      project_scope.first_by_external_id(namespace: SYSTEM_ID, value: mper_id)
+    end
+
+    # @param mper_id [String]
+    # @return [Hmis::UnitType, nil]
+    def find_unit_type_by_mper(mper_id)
+      ::Hmis::UnitType.first_by_external_id(namespace: SYSTEM_ID, value: mper_id)
+    end
+
+    # @param source [ApplicationRecord]
+    # @return [String, nil]
+    def identify_source(source)
+      external_ids.where(source: source).first&.value
+    end
+
+    # @param source [ApplicationRecord]
+    # @param value [String]
+    # @return [HmisExternalApis::ExternalId]
+    def create_external_id(source:, value:, **attrs)
+      external_ids.create!(source: source, value: value, remote_credential: remote_credential, **attrs)
+    end
```

Comment:
Is there any case where we might be getting more than a handful of these?  If so, I'd like to go ahead and optimize this with a batch import.

Reply:
@eanders since we're not storing MPER Project|Organization|FunderIDs as External IDs anymore, I don't think that's necessary. The only thing we'll have "MPER External IDs" for really is Unit Types (which don't really come from MPER at all).

@ttoomey for context, we decided after all to store the MPER IDs on the Project|Organization|Funder records themselves, in the existing ProjectID etc columns. We already do this in the OP warehouse when we import them from the Allegheny warehouse anyway. This [importer](https://www.pivotaltracker.com/n/projects/2591838/stories/185008935) ticket is updated.